### PR TITLE
Change stale threshold

### PR DESCRIPTION
**Pull request by Pandapip1**
See #5360

### CODE REVIEW

**Comment by MicahZoltu** [comment]
As mentioned in the previous PR, I'm generally in favor of this but I would like to wait to hear from other editors before merging, or discussing it at fortnightly meeting.  Hopefully we can get some feedback relatively quickly, but if not then we can wait 3-4 weeks to merge like usual.

**Comment by Pandapip1** [comment]
I had (again incorrectly) assumed you had approved with your comment. I guess I will wait for an _explicit_ review with approval status. @SamWilsn @gcolvin @axic @lightclient if you approve of any of the CI-changing PRs, please **do** explicitly approve the PR to signal your approval (otherwise, we have no way of knowing).

**Comment by Pandapip1** [comment]
This will need the `waiting: no recent activity` label.

**Comment by Pandapip1** [comment]
The repository, not the PR.

**Comment by skaille** [comment]
gh pr checkout 5393

File: .github/workflows/stale.yml
Comment (SamWilsn) [inline .github/workflows/stale.yml:28]:
```suggestion
        stale-issue-label: "waiting: no recent activity"
```

Yay? Nay?

Comment (Pandapip1) [reply .github/workflows/stale.yml:28]: Yay.

**Comment by xinbenlv** [comment]
I think 7days to call an issue stale is a bit extreme. By that standard, this issue have been stale.

I'd like to keep it 90-days or at least 60 days to welcome contributor input.

**Comment by lightclient** [comment]
I think 3-6 months before marking stale is fine then give 1-2 weeks to respond.

**Comment by Pandapip1** [comment]
@SamWilsn merge?

**Comment by Pandapip1** [comment]
> I think 7 days to call an issue stale is a bit extreme. By that standard, this issue has been stale.

That's the point. It keeps PR authors engaged in the process, to re-request editors for review or to remind them to fix stuff.

It won't be *closed* until 3 months have passed. Changes can still be made in that time, and that will reset the clock.

**Comment by xinbenlv** [comment]
Got it. For mercy, can we say "stale" in <s>1</> **2** weeks then close in 3 month?

**Comment by Pandapip1** [comment]
I'd say 1 week is the sweet spot. If neither an editor nor an author has touched something for one week, it's probably best to give them a prod.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**
